frontend/components/Nav.vue

```diff
@@ -4,13 +4,29 @@
       <div class="flex justify-between h-16">
         <div class="flex">
           <div class="noselect flex-shrink-0 flex items-center">
-            <router-link to="/home">
-              <a class="logo noselect" title="Home" @click="goHome()"
-                >Znajdź Praktyki</a
-              >
+            <router-link to="/">
+              <a class="logo noselect" title="Home">Znajdź Praktyki</a>
             </router-link>
```

Comment:
Why do u need `<a>` inside `router-link`? It's redundant.

Reply:
Oh yeah, it wasn't coded by me so I didn't pay attention to that. I will change it, thanks!